src/static/css/base.sass

```diff
@@ -6,34 +6,39 @@
 
 h1, h2, h3, h4, h5, h6
   text-align: left
+  line-height: 1.3em
+  margin-bottom: 15px
+
+h1
+  font-size: 42px
+
 
 h2
-  font-size: 36px
+  font-size: 33px
   font-weight: 300
-  margin-top: 35px
-  margin-bottom: 30px
+  margin-top: 45px
+  line-height: 1.45em
```

Comment:
Why not have line heights directly related to the element's font size, rather than related to the root font size? It would be much easier to adjust in the future.

You can do that by setting a number to the `line-height` property, e. g.:

```css
font-size: 16px;

/* This will have the same value as the font size = 16px */
line-height: 1; 

/* This will have the font size * 1.2 = 19,2px */
line-height: 1.2; 
```

Reply:
🤔 

I thought thats what `em` did?  https://www.w3schools.com/cssref/css_units.asp

> em: Relative to the font-size of the element (2em means 2 times the size of the current font)




For root size it would be `rem`. 

Not sure what happens when you leave out the unit altogether, maybe it indeed defaults to `em`?

Reply:
Yeah, you’re right. I was mistaking `em` for `rem` 🤦‍♂️
For line-heights, leaving out the unit is the same as using `rem`